substrate-node/pallets/pallet-smart-contract/src/storage_state/v8.rs

```diff
@@ -0,0 +1,1047 @@
+        "🏁  Smart Contract pallet {:?} checking ContractBillingInformationByID storage map END",
+        PalletVersion::<T>::get()
+    );
+
+    T::DbWeight::get().reads_writes(r + 2, 0)
+}
+
+// NodeContractResources
+pub fn check_node_contract_resources<T: Config>() -> frame_support::weights::Weight {
+    info!(
+        "🔎  Smart Contract pallet {:?} checking NodeContractResources storage map START",
+        PalletVersion::<T>::get()
+    );
+
+    let mut r = 0;
+
+    for (contract_id, contract_resource) in NodeContractResources::<T>::iter() {
+        r += 1;
+
+        if contract_resource.contract_id != contract_id {
+            info!(
+                " ⚠️    NodeContractResources[contract: {}]: wrong contract id on resource ({})",
+               contract_id, contract_resource.contract_id
+            );
+        }
+
+        if let Some(c) = Contracts::<T>::get(contract_id) {
+            match c.contract_type {
+                types::ContractData::NodeContract(_) => (),
+                _ => {
+                    info!(
+                        " ⚠️    NodeContractResources[contract: {}]: contract is not a node contract",
+                        contract_id
+                    );
+                }
+            }
+        } else {
+            info!(
+                " ⚠️    NodeContractResources[contract: {}]: contract not exists",
+                contract_id
+            );
+        }
+        r += 1;
+    }
+
+    info!(
+        "🏁  Smart Contract pallet {:?} checking NodeContractResources storage map END",
+        PalletVersion::<T>::get()
+    );
+
+    T::DbWeight::get().reads_writes(r + 2, 0)
+}
+
+// Contracts
+pub fn clean_contracts<T: Config>() -> frame_support::weights::Weight {
+    info!(
+        "🧼  Smart Contract pallet {:?} cleaning Contracts storage map START",
+        PalletVersion::<T>::get()
+    );
+
+    let mut r: u64 = 0;
+    let mut w: u64 = 0;
+
+    let contracts = Contracts::<T>::iter().collect::<Vec<_>>();
+    r += contracts.len() as u64;
+
+    for (contract_id, contract) in contracts.into_iter() {
+        match contract.contract_type {
+            types::ContractData::NodeContract(node_contract) => clean_node_contract::<T>(
+                node_contract.node_id,
+                contract_id,
+                node_contract.deployment_hash,
+                &mut r,
+                &mut w,
+            ),
+            types::ContractData::NameContract(_) => clean_name_contract::<T>(),
+            types::ContractData::RentContract(_) => clean_rent_contract::<T>(),
+        }
+
+        // ContractLock
+        if !ContractLock::<T>::contains_key(contract_id) {
+            let now = <timestamp::Pallet<T>>::get().saturated_into::<u64>() / 1000;
+            r += 1;
+            let mut contract_lock = types::ContractLock::default();
+            contract_lock.lock_updated = now;
+            ContractLock::<T>::insert(contract_id, contract_lock);
+            w += 1;
+        }
```

Comment:
Not sure it is the right thing to do here, in case there is no contract lock associated to a contract